res/css/views/rooms/_E2EIcon.pcss

```diff
@@ -52,17 +52,16 @@ Please see LICENSE files in the repository root for full details.
 
 .mx_E2EIcon_normal::after {
     mask-image: url("$(res)/img/e2e/normal.svg");
-    background-color: var(--cpd-color-icon-tertiary);
+    background-color: white;
 }
 
 .mx_E2EIcon_verified::after {
     mask-image: url("$(res)/img/e2e/verified.svg");
     background-color: $e2e-verified-color;
 }
 
-// When using the "normal" icon as a background for verified or warning icon, 
+// When using the "normal" icon as a background for verified or warning icon,
 // it should be slightly smaller than the foreground icon
 .mx_E2EIcon_verified, .mx_E2EIcon_warning .mx_E2EIcon_normal::after {
     mask-size: 90%;
-    background-color: white;
 }
```

Comment:
With the bg color removed, what mask is being downsized to 90%?

Reply:
I removed the mask https://github.com/element-hq/element-web/pull/30299/commits/f1b18980accc114d5c3ab583beb411b495f74639